pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "dcicutils"
-version = "1.4.0"
+version = "1.5.0"
```

Comment:
Does this really merit a minor version bump? I'd think patch is more appropriate since we are strictly adding new functionality.

Reply:
New functionality _is_ a minor version bump. Patches are for no change.